package.json

```diff
@@ -37,7 +37,7 @@
   },
   "scripts": {
     "lint": "eslint .",
-    "test": "./node_modules/.bin/mocha --timeout 5000 -R spec",
+    "test": "./node_modules/.bin/mocha --timeout 9999 -R spec",
```

Comment:
to be precise, i'm backporting the changes in `master` branch.  If `9999` is not ok for timeout, we need to fix for `master` branch too.